packages/chat-client/src/controllers/engine.ts

```diff
@@ -847,10 +885,29 @@ export class ChatEngine extends IChatEngine {
         publicKey: decodedPayload.iss,
       });
 
+      const authorAccount = cacao.p.iss.split(":").slice(2).join(":");
```

Comment:
Can we move this kind of manipulation into core/utils ? I've seen it in a few different places.

Reply:
We definitely need a bunch of address manipulation utils. For these things as well as prefixing chainId etc. I'll spec out an issue soon